README.md

```diff
@@ -155,12 +155,12 @@ exp = Experiment(api_key="YOUR API KEY")
 
 Run your experiment and click on the link in stdout to be taken back to the Comet UI. Automatically you'll see your metrics, hyperparameters, graph definition, system and environment details and more.
 
-To see the carbon footprint of your experiment, go to the `Panels` tab in the left sidebar and click `Add Panel`.
+**Comet** will automatically create an `EmissionsTracker` from the `codecarbon` package. To visualize the carbon footprint of your experiment, go to the `Panels` tab in the left sidebar and click `Add Panel`.
 
 From the Panel Gallery click the `Public` tab and search for `Emissions Tracker`. Once you've found it, add it to your Experiment.
 
 ![](docs/_gifs/panel-gallery.gif)
 
-Now back in the `Panels` tab you'll see your Emissions Tracker visualization in the Comet UI. To render the Emissions Tracker visualization by default, save your `View`. And voilà! Every time you run your experiments, you'll be able to visualize your CodeCarbon emissions data alongside everything else you need to track for your research. 
+Now back in the `Panels` tab you'll see your Emissions Tracker visualization in the Comet UI. To render the Emissions Tracker visualization by default, save your `View`. And voilà! Every time you run your experiments, you'll be able to visualize your CodeCarbon emissions data alongside everything else you need to track for your research.
```

Comment:
Shouldn't the panel also be called code carbon?

Reply:
@dsblank

Reply:
Yeah, but I'd go ahead and merge this PR. We'll probably need to update a few things as we get closer to release.